app/Services/RabbitMQ.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Services;
+
+use Mookofe\Tail\Facades\Tail;
+
+/**
+ * Class RabbitMQ
+ * @package App\Services
+ */
+class RabbitMQ
+{
+    public static function addTask($queue, $payload)
+    {
+        try {
+            Tail::add($queue, $payload);
+        } catch (\Exception $e) {
+            return response()->json($e->getMessage(), 400);
+        }
+
+        return response()->json('Task successfully added to RabbitMQ queue.', 200);
```

Comment:
non model responses should follow format `["messages" => ["<MESSAGE>"]]` and errors `["errors" => ["<ERROR>"]]`